package.json

```diff
@@ -15,14 +15,15 @@
     "@mui/x-data-grid": "^6.18.7",
     "@propelauth/react": "^2.0.18",
     "axios": "^1.6.5",
+    "eslint-config-next": "^13.5.6",
     "eslint-config-react-app": "^7.0.1",
     "moment": "^2.30.1",
-    "next": "^14.0.4",
+    "next": "^13.5.6",
```

Comment:
@Ashmitha-art  hey there - normally we don't downgrade packages, can you share why you did this?

Reply:
Hey, I was having issues with running rpm run build on my MacOS and when I checked online this was one way to run it by downgrading next js, so I had to downgrade to allow it to run locally on my PC.

Reply:
Can you share the error? Curious what issue you had so we can still have a current version - locking to a version is generally not a good practice @Ashmitha-art

Reply:
Error: You are using Node.js 18.16.0. For Next.js, Node.js version >= v18.17.0 is required. 

Updating node js to the latest version also didn't work. 

https://www.reddit.com/r/nextjs/comments/18icsu8/you_are_using_nodejs_1772_for_nextjs_nodejs/

https://www.reddit.com/r/nextjs/comments/18icsu8/comment/khcylj1/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

https://github.com/vercel/next.js/discussions/59518

https://github.com/vercel/next.js/discussions/59518#discussioncomment-8159185